Adding redirect URI to ADO (#6843)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/git/setup-azure-service-principal.md

4. Provide a name for your app. We recommend using, "dbt Labs Azure DevOps app".
5. Select **Accounts in any organizational directory (Any Entra ID directory - Multitenant)** as the Supported Account Types.
Many customers ask why they need to select Multitenant instead of Single Tenant, and they frequently get this step wrong. Microsoft considers Azure DevOps (formerly called Visual Studio) and Microsoft Entra ID separate tenants, and for the Entra ID application to work properly, you must select Multitenant.

> 6. Set **Redirect URI (optional)** to **Web**. Copy the value from dbt Cloud to the next field:

Reviewer: ```suggestion
6. Set **Redirect URI (optional)** to **Web**. Copy and paste the Redirect URI from dbt Cloud into the next field.  To find the Redirect URI in dbt Cloud:
```

    1. In dbt Cloud, navigate to **Account Settings** -> **Integrations**.
    2. Click the **edit icon** next to **Azure DevOps**.

>     3. Copy the first **Redirect URIs** value to the field in Azure (not the value that ends with azure_active_directory_service_user). It will look like `https://<YOUR_ACCESS_URL>/complete/azure_active_directory`.

Reviewer: ```suggestion
    3. Copy the first **Redirect URIs** value which looks like `https://<YOUR_ACCESS_URL>/complete/azure_active_directory` and does NOT end with `service_user`.
```

>     3. Copy the first **Redirect URIs** value to the field in Azure (not the value that ends with azure_active_directory_service_user). It will look something like `https://<YOUR_ACCESS_URL>/complete/azure_active_directory`.

Author: ```suggestion
    3. Copy the first **Redirect URIs** value to the field in Azure (not the value that ends with azure_active_directory_service_user). It will look like `https://<YOUR_ACCESS_URL>/complete/azure_active_directory`.
```
